backend/oasst_backend/config.py

```diff
@@ -20,28 +20,28 @@ class TreeManagerConfiguration(BaseModel):
     """Maximum number of reply messages per tree node."""
 
     goal_tree_size: int = 15
-    """Total number of messages to gather per tree"""
+    """Total number of messages to gather per tree."""
 
     num_reviews_initial_prompt: int = 3
     """Number of peer review checks to collect in INITIAL_PROMPT_REVIEW state."""
 
     num_reviews_reply: int = 3
-    """Number of peer review checks to collect per reply (other than initial_prompt)"""
+    """Number of peer review checks to collect per reply (other than initial_prompt)."""
```

Comment:
(I omitted the full-stop at the end of these comments because they are not full sentences (a verb is missing))